pkg/tsdb/cloudwatch/request_parser_test.go

```diff
@@ -21,15 +22,15 @@ func TestRequestParser(t *testing.T) {
 			}
 			oldQuery.RefID = "A"
 			oldQuery.JSON = []byte(`{
-				"region": "us-east-1",
```

Comment:
These shouldn't have to change to parse successfully, I believe unmarshaling is case insensitive

Reply:
I'll double check!